webcompat/templates/shared/svg-icons.html

```diff
@@ -0,0 +1,316 @@
+<svg class="hide">
```

Comment:
This would cause the svg to be visible! The `.hide` class is only available in the style guide, it's not exported into the CSS we use on webcomapt.com. Therefore, use `.is-hidden` please.